### PR TITLE
Fix EGW azure docs

### PR DESCRIPTION
**Pull request by sridhartigera**
Product Version(s):
Calico-enterprise - v3.16
Calico-cloud

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Review by fasaxc — APPROVED** [review]

File: calico-cloud/networking/egress/egress-gateway-azure.mdx
Comment (ctauchen) [inline calico-cloud/networking/egress/egress-gateway-azure.mdx:1]: Are we sure this is going in CC? I thought we'd been through this one already. @sridhartigera

**Review by ctauchen — COMMENTED** [review]
LGTM, pending OK from @tmjd on the CC-ification of the Azure doc.

**Comment by ctauchen** [comment]
@tmjd Can you take a look at this for CC? Previously, you said that the EGW on Azure doc wouldn't work for CC: https://github.com/tigera/docs/pull/422#discussion_r1128660112

Here, @sridhartigera has changed the calcoctl commands to kubectl. Is that enough? Are you happy for this to merge?

File: calico-cloud/networking/egress/egress-gateway-azure.mdx
Comment (tmjd) [reply calico-cloud/networking/egress/egress-gateway-azure.mdx:1]: I think adding it for CC next is fine. I checked with @Glen-Tigera and he said it has been tested and is working.

**Review by tmjd — COMMENTED** [review]
I think my overall question is why include the removal of the `.p` from the felixconfiguration resource type. I think it was previously a precaution to ensure the correct resource was being updated.
I don't know if we should maybe keep it because I think it would ensure there was no chance that the CRD type would be updated rather than the projectcalico.org type.
Was that change discussed by the broader team?

File: calico-cloud/networking/egress/egress-gateway-azure.mdx
Comment (ctauchen) [reply calico-cloud/networking/egress/egress-gateway-azure.mdx:1]: Ok, thanks!

**Comment by sridhartigera** [comment]
> I think my overall question is why include the removal of the `.p` from the felixconfiguration resource type. I think it was previously a precaution to ensure the correct resource was being updated. I don't know if we should maybe keep it because I think it would ensure there was no chance that the CRD type would be updated rather than the projectcalico.org type. Was that change discussed by the broader team?

I had a discussion with @fasaxc and he was ok with changing ```felixconfiguration.p``` to ```felixconfiguration```.

**Review by tmjd — APPROVED** [review]

File: calico-cloud/networking/egress/egress-gateway-azure.mdx
Comment (tmjd) [inline calico-cloud/networking/egress/egress-gateway-azure.mdx:195]: This command still does not work for clusters that originally started from an AKS cluster that had Calico managed by AKS. But this is also a problem with the Enterprise documentation.

Comment (tmjd) [reply calico-cloud/networking/egress/egress-gateway-azure.mdx:195]: Here was my original comment about this too https://github.com/tigera/docs/pull/422#discussion_r1128660579

Comment (tmjd) [reply calico-cloud/networking/egress/egress-gateway-azure.mdx:195]: Since this doc requires "Calico CNI", this comment isn't applicable for this doc.

**Pull request merged**
